laser_scan_matcher/src/laser_scan_matcher.cpp

```diff
@@ -162,6 +166,12 @@ void LaserScanMatcher::initParams()
   if (!nh_private_.getParam ("use_vel", use_vel_))
     use_vel_ = false;
 
+  // **** Are velocity input messages stamped?
+  // if false, will subscribe to Twist msgs on /vel
+  // if true, will subscribe to TwistStamped msgs on /vel
+  if (!nh_private_.getParam ("stamped_vel", stamped_vel_))
```

Comment:
I added a parameter in my launch file. As by default it expects stamped Twist messages, nobody will notice the change unless adding stamped_vel as true in the launch file